tools/clang/test/CodeGenSPIRV/method.byte-address-buffer.load.layout.hlsl

```diff
@@ -0,0 +1,23 @@
+// RUN: %dxc -spirv -T lib_6_5 -E AnyHit1 -fspv-target-env=vulkan1.3 -Wno-effects-syntax -fcgl -Vd %s | FileCheck %s
```

Comment:
`-Vd`, but wasn't the bug a validation error? Is the test failing without the fix?

Reply:
Good catch. I cut & paste the command I was using, and I turned off validatoin so I could see the code that was generated.